cli/src/commands/content/createVideo.ts

```diff
@@ -0,0 +1,64 @@
+import ContentDirectoryCommandBase from '../../base/ContentDirectoryCommandBase'
+import { IOFlags, getInputJson, saveOutputJson } from '../../helpers/InputOutput'
+import { videoMetadataFromInput } from '../../helpers/serialization'
+import { VideoCreationParameters, VideoCreationParametersInput } from '../../Types'
+
+export default class CreateVideoCommand extends ContentDirectoryCommandBase {
+  static description = 'Create video under specific channel inside content directory.'
+  static flags = {
+    context: ContentDirectoryCommandBase.contextFlag,
+    ...IOFlags,
+  }
+
+  static args = [
+    {
+      name: 'channelId',
+      required: true,
+      description: 'ID of the Channel',
+    },
+  ]
+
+  async run() {
+    let { context, input, output } = this.parse(CreateVideoCommand).flags
+
+    const { channelId } = this.parse(CreateVideoCommand).args
+
+    if (!context) {
+      context = await this.promptForContext()
```

Comment:
Good point, did not thought about that before, but it would not possible to convert owner into actor, because channel owner Curator representation does not hold curator_id to perform authorization. https://github.com/Joystream/joystream/blob/sumer/runtime-modules/content/src/permissions/mod.rs#L153 
Reworked to prompt for owner context, though. https://github.com/Joystream/joystream/pull/2262/commits/d7ffaae4490a4ef514cc6a3cbb22175d63678089